### ISSUE TITLE
Torrent download filters are not wokring correctly

### ISSUE THREAD

**Issue by Tymec** [erengy/taiga#646]
So I have those filters, but it still downloads sometimes the same anime episode but with different resolutions (480p, 720p and 1080p), even though I have a filter to only download 1080p. Same with the Fansub filter, I have selected HorribleSubs, yet it downloads some episodes from EraiRaws.
![settings](https://user-images.githubusercontent.com/2727109/42395475-5cba0894-815e-11e8-8a9d-719316237aaf.png)

**Comment by erengy** [comment]
Please export your filters (using the rightmost button at *Settings → Torrents → Filters*) so I can take a look.

**Comment by Tymec** [comment]
This right?
+WlmRhzav1yJIzMjSgS5bHU6u4IJMeuvqejcntpPyvfW/fjyi8m05vpt3X5qcpKNzgN/hPOFzylw==

**Comment by erengy** [comment]
Yes, but please copy the entire text, not just the portion you see on the screen.

**Comment by Tymec** [comment]
Oh shit, sorry didn't realize it was just a portion. Here:
VEFJAQUAMS4zLjByAaUFeJy9lFtLw0AQhZ/trxgW+lhaBa1CUxBEfBTqm0iYJNN2YC9xL5H+ezdJL/qQVC31KZA5M/udOcvO2JMCzD0bnQhHknIvQKHP14lAvRFgyrZW0BKDjEXSmEkqEuFtIAEaFSVi0XRCHqwl7eUGPuoRrFdiPriY5UYXXI+BKFNRkIjgyKbOow+uPoMsemMTQe8BZfxToQxx7KWA8XwwG9eU8dsLK+UvYZFtBATUrAhYQylRgzctehe3Io/Hua8a7hOMX/cYL9jlaIuumGpZhZ46zD+03dF2AQcxaONHrEeSnW83clpwk7/z9yW3g28iotYEVsiy1gKV7ExBrgt9W091UBnZr/SS9uTD7exhd4TS5CjT3bT9+T3reGk8nOEqvz6idiF7g/u+zHaoK2tC2YP5ZKzleMgiZO4suM+WlmRhzav1yJIzMjSgS5bHU6u4IJMeuvqejcntpPyvfW/fjyi8m05vpt3X5qcpKNzgN/hPOFzylw==

**Comment by Tymec** [comment]
Thanks a lot! Do you have any idea about the qBitTorrent thing or is it something to do with qBitTorrent itself. It puts unfinished downloads into the temp folder and moves them when they are finished, but the downloads from Taiga seem to always be unfinished.

**Comment by erengy** [comment]
Do you have Taiga's *Use anime folders as the download folder* option enabled? If it's not, it's unlikely to be related to Taiga. In any case, you can try enabling/disabling that option and see if it makes any difference.
